indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -290,6 +299,52 @@ class BookRoomModal extends React.Component {
         );
     }
 
+    renderEventItem = (event) => {
+        const start = `${moment(event.startDt).format('L')} ${moment(event.startDt).format('LT')}`;
+        const sameDate = moment(event.startDt).isSame(event.endDt, 'date');
+        const endTime = moment(event.endDt).format('LT');
+        const end = sameDate ? `${endTime}` : `${moment(event.endDt).format('L')} ${endTime}`;
+        return {
+            text: event.title,
+            description: `${start} - ${end}`,
+            value: event.id
+        };
+    };
+
+    renderRelatedEventsDropdown = (disabled) => {
+        const {relatedEvents} = this.props;
+        const options = relatedEvents.map(this.renderEventItem);
+
+        if (!relatedEvents.length) {
+            return;
+        }
+
+        return (
+            <Segment>
+                <h3><Icon name="chain" />{Translate.string('Event')}</h3>
+                <div styleName="events-segment-description">
+                    <PluralTranslate count={relatedEvents.length}>
+                        <Singular>
+                            You have an event taking place in this room overlapping with booking times.
+                            If you are booking this room for this event, please select it below.
```

Comment:
I think "...the room for this event..." sounds better than having "this" there twice